src/main/java/controllers/PostController.java

```diff
@@ -115,7 +115,13 @@ private String getMeasurableIngredients(String promptMeasurable) {
             } else if (strippedIngredient.split(" ").length < 3) {
                 inOut.setOutput(errorMessage);
                 measurableInput = getMeasurableIngredients(promptMeasurable);
-            }
+            } /*
```

Comment:
Why did you add a comment. You could delete this part right?

Reply:
the comment continues after that line; it's a comment regarding another bug I found which I don' know how to solve yet